backend/app.py

```
@@ -23,7 +23,7 @@ def login():
     hashed = hashlib.md5(password.encode("utf-8")).hexdigest()
```

Ruff (0.13.2)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)